react/features/display-name/components/DisplayName.web.js

```diff
@@ -144,6 +155,7 @@ class DisplayName extends Component<Props, State> {
         if (allowEditing && this.state.isEditing) {
             return (
                 <input
+                    autoComplete = 'off'
```

Comment:
why? browsers usually remember your name and surname so it can be handy, can't it?